demo/package.json

```diff
@@ -16,11 +16,15 @@
     "tns-core-modules": "^4.2.0"
   },
   "devDependencies": {
-    "nativescript-dev-typescript": "^0.7.2",
-    "nativescript-dev-webpack": "~0.15.1",
-    "typescript": "~2.8.2",
+    "babel-traverse": "6.26.0",
```

Comment:
I think the changes to this file are not needed and it should not be committed.

Reply:
Reverted to original